parsons/notifications/slack.py

```diff
@@ -86,6 +91,34 @@ def users(self, fields=['id', 'name', 'deleted', 'profile_real_name_normalized',
 
         return tbl
 
+    @classmethod
+    def message(cls, channel, text, webhook=None, api_key=None, parent_message_id=None):
```

Comment:
I think in the name of avoiding having multiple ways to do the same thing, could you remove the `api_key` fallback?

Reply:
done!